components/match2/commons/starcraft_starcraft2/opponent_display_starcraft.lua

```diff
@@ -241,6 +241,28 @@ end
 
 StarcraftOpponentDisplay.CheckMark = '<i class="fa fa-check forest-green-text" aria-hidden="true"></i>'
 
+StarcraftOpponentDisplay.propTypes.BlockScore = {
+	isWinner = 'boolean?',
+	scoreText = 'any',
+	side = 'string',
```

Comment:
For general block display components I think it's better to use `flip=true/false` instead of side=`'left'/'right'`. The reason is that there are legitimate use cases for flipped component to appear on the right side (and non-flipped on the left)

For example both of these look ok from a design perspective
![image](https://user-images.githubusercontent.com/85348434/141672230-68b970bf-9f8e-4ae4-b062-e733188fec41.png)
![image](https://user-images.githubusercontent.com/85348434/141672328-64add98a-fb82-4ffc-a91d-bbc5bb1f2d97.png)

Reply:
well the `'left'` one adds a border to its right while the `'right'` one adds a border to its left (via the css class)
adjusted it a bit, please check if you like this more^^